sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -18,6 +18,7 @@
  */
 public class DefaultAzureCredentialBuilder extends CredentialBuilderBase<DefaultAzureCredentialBuilder> {
     private String tenantId;
+    private String managedIdentityClientId;
```

Comment:
Shouldn't this default to the value in `AZURE_CLIENT_ID` if set?

Reply:
The tenant ID isn't AZURE_TENANT_ID by default, would this cause confusion?

Reply:
It looks like all environment variables are checked in .NET - adding to Java too

Reply:
Updated